docker-compose.yml

```diff
@@ -3,25 +3,39 @@ version: '3'
 services:
   arangodb:
     image: arangodb:3.5.1
+    restart: always
+    ports:
+      - 8529:8529
     environment:
       ARANGO_NO_AUTH: 1
 
   spline:
-    image: absaoss/spline-rest-server
+    image: "absaoss/spline-rest-server:0.5.1"
```

Comment:
Is it possible to use a variable instead of a hardcoded version?
Something like `absaoss/spline-rest-server:${spline_version}`

Reply:
it's possible to use environment variables but AFAIK there's no way to define default values for them

so instead of
`docker-compose up`
everyone would need to call
`spline_version=0.5.1 docker-compose up` 
not sure if it's a way to go ?

Reply:
What about `.env`-file?
https://docs.docker.com/compose/environment-variables/#the-env-file

Reply:
yep that's an option I'll update it

Reply:
Done